src/shapley_sensitivity.jl

```diff
@@ -168,23 +151,18 @@ function cond_sampling(distribution::SklarDist,
     n_dep = length(idx)
 
     # need to sample from univariate normal and multivariate normal using different functions
+    samples = zeros(eltype(cond_mean), n_dep, n_sample)
     if n_dep == 1
-        dist_cond = Normal(cond_mean[1, 1], cond_var[1, 1])
-        sample_norm = rand(dist_cond, n_sample)
+        dist_cond = Normal(cond_mean[1, 1], sqrt(cond_var[1, 1]))
```

Comment:
I didn't rerun the tests locally... I wonder if the test errors are caused by this fix (stddev instead of variance).

Reply:
yes that should be it. Fwiw the values for ishigami function are closer to the sobol indices now so it's not suspicious

Reply:
closer in a very very handwavy way lol